sdk/keyvault/Azure.Security.KeyVault.Administration/tests/samples/SampleSnippets.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.Security.KeyVault.Administration.Models;
+using Azure.Security.KeyVault.Tests;
+using NUnit.Framework;
+
+namespace Azure.Security.KeyVault.Administration.Samples
+{
+    /// <summary>
+    /// Samples that are used in the associated README.md file.
+    /// </summary>
+    public partial class Snippets : SamplesBase<KeyVaultTestEnvironment>
```

Comment:
This should inherit from our existing base class, and create the client from the base method which instruments it so we can use recorded tests, ideally. See the other libraries' samples for an example.